expression/builtin_string_vec_test.go

```diff
@@ -29,8 +29,14 @@ var vecBuiltinStringCases = map[string][]vecExprBenchCase{
 	ast.Convert:        {},
 	ast.Substring:      {},
 	ast.SubstringIndex: {},
-	ast.Locate:         {},
-	ast.Hex:            {},
+	ast.Locate: {
+		{
+			retEvalType:   types.ETInt,
+			childrenTypes: []types.EvalType{types.ETString, types.ETString, types.ETInt},
+			geners:        []dataGenerator{&randLenStrGener{0, 10}, &randLenStrGener{0, 20}, &rangeInt64Gener{-10, 20}},
```

Comment:
How about adding one more case where the length of `substrs` is between `1` to `2` to increase the probability that the `substr` is included in the `str`.